packages/core/src/utils/fileUtils.ts

```diff
@@ -14,9 +14,9 @@ export function glob(expression: string): Promise<string[]> {
 }
 
 export function deleteDir(dirToDelete: string): Promise<void> {
-  return new Promise<void>((resolve, reject) => {
+  return new Promise<void>((_, reject) => {
```

Comment:
So this promise is never resolved?

Reply:
I think it should not resolve like this at least. From what i could see Error callback always return Error so check error ? reject() : resolve()  will always return reject()